lbrynet/dht/msgformat.py

```diff
@@ -43,10 +44,13 @@ def toPrimitive(self, message):
 class DefaultFormat(MessageTranslator):
     """ The default on-the-wire message format for this library """
     typeRequest, typeResponse, typeError = range(3)
-    headerType, headerMsgID, headerNodeID, headerNodeIP, headerNodePort = range(5)
-    headerPayload, headerArgs = range(5, 7)
+    headerCompVer, headerType, headerMsgID, headerNodeID, headerNodeIP, headerNodePort = range(6)
+    headerPayload, headerArgs = range(6, 8)
 
     def fromPrimitive(self, msgPrimitive):
+        if msgPrimitive[self.headerCompVer] != COMPAT_VERSION:
+            return
+
```

Comment:
is "return" the proper thing to do here? 
Maybe we should be throwing a exception and catching it here:
https://github.com/lbryio/lbry/blob/master/lbrynet/dht/protocol.py#L206

Reply:
Yeah, that would be more inline with our coding style.